.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Cypress Tests
+
+on: [push]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v4.2.0 # use the explicit version number
+        with:
+          build: npm run build
+          start: npm start
```

Comment:
💥 Now tests should be pass on Github Action because `cy.visit('http://localhost:4200/')` this line continously failed. There wasn't any local url available.